tests/lib/cli-engine/config-array-factory.js

```diff
@@ -1515,6 +1515,23 @@ describe("ConfigArrayFactory", () => {
             }, /Failed to load config "plugin:invalid-config\/bar" to extend from./u);
         });
 
+        it("should throw an error with a message template when a plugin referenced for a plugin config is invalid", () => {
+            try {
+                applyExtends({
+                    extends: "plugin:nonexistent-plugin",
```

Comment:
Can we rename `nonexistent-plugin` to something else, since this isn't testing a case where the plugin doesn't exist (it would be the same if it does exist)